test/tesla/middleware/compression_test.exs

```diff
@@ -114,7 +116,8 @@ defmodule Tesla.Middleware.CompressionTest do
       {status, headers, body} =
         case env.url do
           "/" ->
-            {200, [{"content-type", "text/plain"}, {"content-encoding", "gzip"}], env.headers}
+            {200, [{"content-type", "text/plain"}, {"content-encoding", "gzip"}],
+             TestSupport.gzip_headers(env)}
```

Comment:
Unclear what this function performs

Reply:
As I said before, the whole testing environment needs some cleanup; it is OK for now.